lms/templates/ccx/underscore/form.underscore

```diff
@@ -0,0 +1,181 @@
+<div class="field">
```

Comment:
See #11899-- some of the templates may have been fixed up already (but then perhaps some changes lost in the rebase?).